app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -69,96 +101,118 @@ def handle_bad_request_error(err, enrollment)
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_exception(
+      **enrollment_attributes(enrollment, false),
       reason: 'Unsupported status',
-      enrollment_id: enrollment.id,
-      enrollment_code: enrollment.enrollment_code,
       status: status,
     )
   end
 
-  def handle_unsupported_id_type(enrollment, primary_id_type)
-    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_failure(
+  def handle_unsupported_id_type(enrollment, response)
+    enrollment_outcomes[:enrollments_failed] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
+      **enrollment_attributes(enrollment, true),
+      fraud_suspected: response['fraudSuspected'],
+      passed: false,
+      primary_id_type: response['primaryIdType'],
       reason: 'Unsupported ID type',
-      enrollment_id: enrollment.id,
-      enrollment_code: enrollment.enrollment_code,
-      primary_id_type: primary_id_type,
     )
+    enrollment.update(status: :failed)
+  end
+
+  def handle_expired_status_update(enrollment)
+    enrollment_outcomes[:enrollments_expired] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
+      **enrollment_attributes(enrollment, true),
+      fraud_suspected: nil,
+      passed: false,
+      reason: 'Enrollment has expired',
+    )
+    enrollment.update(status: :expired)
   end
 
   def handle_failed_status(enrollment, response)
-    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_failure(
-      reason: 'Failed status',
-      enrollment_id: enrollment.id,
-      enrollment_code: enrollment.enrollment_code,
+    enrollment_outcomes[:enrollments_failed] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
+      **enrollment_attributes(enrollment, true),
       failure_reason: response['failureReason'],
       fraud_suspected: response['fraudSuspected'],
+      passed: false,
       primary_id_type: response['primaryIdType'],
       proofing_state: response['proofingState'],
+      reason: 'Failed status',
       secondary_id_type: response['secondaryIdType'],
       transaction_end_date_time: response['transactionEndDateTime'],
       transaction_start_date_time: response['transactionStartDateTime'],
     )
+
+    enrollment.update(status: :failed)
+    send_failed_email(enrollment.user, enrollment)
   end
 
-  def handle_successful_status_update(enrollment)
-    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_success(
+  def handle_successful_status_update(enrollment, response)
+    enrollment_outcomes[:enrollments_passed] += 1
+    analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
+      **enrollment_attributes(enrollment, true),
+      fraud_suspected: response['fraudSuspected'],
+      passed: true,
```

Comment:
It wasn't explicitly mentioned in the ticket (and as such would be fine to consider out-of-scope for this work), but I'm wondering if should be more permissive / consistent with logging details from the response. In particular, I could see the question of "What types of primary IDs are most successful?" being hard to answer here, since we're only logging those response properties in certain events. I'd wonder if a method like `response_attributes` (similar to `enrollment_attributes`) would be useful to create some consistency with how we're logging response properties, at least for the valid responses.